src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  background-color: #f4f4f4;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You should add a fallback font in case the main one doesn't work. For instance, you could modify this line to: `font-family: Roboto, Arial, sans-serif, Helvetica;`